### PR TITLE
[Rename classes that under module adaptive-media, analytics, archived, asset]Lps 166967

### PR DESCRIPTION
**Pull request by slnn**
https://issues.liferay.com/browse/LPS-166967

### CODE REVIEW

**Comment by Kvale1733** [comment]
@slnn No regressions found.

File: modules/apps/adaptive-media/adaptive-media-image-impl-test/src/testIntegration/java/com/liferay/adaptive/media/image/internal/scaler/test/AMImageScalerRegistryImplTest.java [hunk 1]
```diff
@@ -347,7 +348,7 @@ private void _unregisterAMImageScaler(
 	private AMImageScaler _amDefaultImageScaler;
 
 	@Inject
-	private AMImageScalerTracker _amImageScalerTracker;
+	private AMImageScalerRegistry _amImageScalerRegistry;
```
Comment (yuhai) [inline modules/apps/adaptive-media/adaptive-media-image-impl-test/src/testIntegration/java/com/liferay/adaptive/media/image/internal/scaler/test/AMImageScalerRegistryImplTest.java:351]: @slnn, could you please merge the commit to https://github.com/liferay-core-infra/liferay-portal/pull/2299/commits/ea94f990ebc22583f63d13e3d641308eabf46b3c. I think we may treat the change as usage. After that, the pr should be more clear(The first commit: rename *Registry and *RegistryImpl; The second commit: Apply usages).  You may push force your change. And then I will approve the pr to Dante review.

Thanks,
Hai

Comment (slnn) [reply modules/apps/adaptive-media/adaptive-media-image-impl-test/src/testIntegration/java/com/liferay/adaptive/media/image/internal/scaler/test/AMImageScalerRegistryImplTest.java:351]: Done

**Review by yuhai — APPROVED** [review]
@dantewang, the code looks good to me.

**Comment by slnn** [comment]
Forced push to fix the conflicts for bnd and packageinfo files

**Comment by dantewang** [comment]
ci:forward

**Comment by dantewang** [comment]
Forwarding this one @shuyangzhou @tinatian

**Comment by dantewang** [comment]
@slnn Looks like there's new sem ver error? Also, I noticed that you modified classes in `archived`, please exclude those.

**Comment by slnn** [comment]
ci:test:relevant

**Comment by slnn** [comment]
> @slnn Looks like there's new sem ver error? Also, I noticed that you modified classes in `archived`, please exclude those.

Done, forced push to this pull after excluding these ones in  `archived` and rerunning gradlew baseline to update these versions

**Comment by dantewang** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
@slnn no regressions found.

**Comment by dantewang** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/125885

**Pull request closed without merging**